partner_firstname/models/base_config_settings.py

```diff
@@ -62,7 +62,7 @@ def action_recalculate_partners_name(self):
         )
         partners = self._partners_for_recalculating()
         _logger.info("Recalculating names for %d partners.", len(partners))
-        partners._compute_name()
+        self.env.add_to_compute(self.env["res.partner"]._fields["name"], partners)
```

Comment:
A comment explaining why computation is delayed instead of manually calling compute method would be great.